cmd/tdexd/main.go

```diff
@@ -59,6 +59,7 @@ var (
 	rescanRangeStart              = config.GetInt(config.RescanRangeStartKey)
 	rescanGapLimit                = config.GetInt(config.RescanGapLimitKey)
 	walletUnlockPasswordFile      = config.GetString(config.WalletUnlockPasswordFile)
+	runOnOnePort                  = config.GetBool(config.RunOnOnePortKey)
```

Comment:
what if we infer this when trade and operator port are equal?

Reply:
Done